DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    f"Datalink link service url: {dataLinkUrl}"
</pre><p>When I executed this, it didn't print the URL as intended. I think it's missing <code>print()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Yeah seems like this only works if its the last line of a cell (see comment above; maybe it otherwise gets overwritten? I'm not sure. f" " should print.) In any case I have rewritten so it now prints.</p>